website/static/js/comment.js

```diff
@@ -298,7 +298,12 @@ var CommentModel = function(data, $parent, $root) {
             'gravatarUrl': userData.links.profile_image
         };
     } else {
-        self.author = self.$root.author;
+        self.author = {
+            'id': null,
+            'urls': {'profile': ''},
+            'fullname': 'Deactivated user',
```

Comment:
Still, sufficient `meta` information is returned for proper display:

<img width="295" alt="screen shot 2016-05-23 at 13 11 53" src="https://cloud.githubusercontent.com/assets/5659262/15478401/fff98220-20e7-11e6-9ce9-f2b7b1a7e4f3.png">